utils/tasks/docker-run-amd64.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+
+# Run the local docker image, see the Taskfile.yml for usage.
+
+if [ $(uname -m) == 'arm64' ]; then
```

Comment:
FYI I tried to run this from main and it did not detect the platform properly here:
```
task test-docker
task: [test-docker] task docker-run -- task test
task: [docker-build] ./utils/tasks/docker-build.sh
+ source utils/tasks/docker-setup.sh
++ set -x
+++ uname -m
++ '[' x86_64 == arm64 ']'
```
I remember it was running under Rosetta when launched from pip environment. Not sure if it got fixed here.